[2024-03-18T08:04:36.413+0000] {processor.py:161} INFO - Started process (PID=5094) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:04:36.415+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:04:36.416+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:04:36.416+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:04:36.789+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:04:36.800+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.392 seconds
[2024-03-18T08:04:48.420+0000] {processor.py:161} INFO - Started process (PID=5228) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:04:48.421+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:04:48.423+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:04:48.422+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:04:48.723+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:04:48.835+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:04:48.834+0000] {override.py:1769} INFO - Created Permission View: can edit on DAG:weather_dag
[2024-03-18T08:04:48.852+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:04:48.852+0000] {override.py:1769} INFO - Created Permission View: can delete on DAG:weather_dag
[2024-03-18T08:04:48.864+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:04:48.864+0000] {override.py:1769} INFO - Created Permission View: can read on DAG:weather_dag
[2024-03-18T08:04:48.865+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:04:48.864+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:04:48.877+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:04:48.876+0000] {dag.py:3069} INFO - Creating ORM DAG for weather_dag
[2024-03-18T08:04:48.894+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:04:48.893+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-02-28 00:00:00+00:00, run_after=2024-02-29 00:00:00+00:00
[2024-03-18T08:04:48.919+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.502 seconds
[2024-03-18T08:05:41.730+0000] {processor.py:161} INFO - Started process (PID=5758) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:05:41.732+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:05:41.734+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:05:41.733+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:05:42.077+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:05:42.092+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:05:42.091+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:05:42.114+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:05:42.114+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-02-28 00:00:00+00:00, run_after=2024-02-29 00:00:00+00:00
[2024-03-18T08:05:42.144+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.417 seconds
[2024-03-18T08:06:53.741+0000] {processor.py:161} INFO - Started process (PID=6638) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:06:53.743+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:06:53.744+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:06:53.743+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:06:53.748+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:06:53.747+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 8
    from airflow.operators
                          ^
SyntaxError: invalid syntax
[2024-03-18T08:06:53.749+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:06:53.771+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.034 seconds
[2024-03-18T08:07:41.970+0000] {processor.py:161} INFO - Started process (PID=7140) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:07:41.972+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:07:41.973+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:07:41.973+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:07:42.297+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:07:42.294+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 93, in <module>
    transform_load_weather_data = PythonOperator(
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 420, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-03-18T08:07:42.298+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:07:42.319+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.352 seconds
[2024-03-18T08:08:30.925+0000] {processor.py:161} INFO - Started process (PID=7370) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:08:30.927+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:08:30.928+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:08:30.927+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:08:31.243+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:08:31.242+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 8, in <module>
    from airflow.providers.python.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.python'
[2024-03-18T08:08:31.244+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:08:31.264+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.343 seconds
[2024-03-18T08:09:15.321+0000] {processor.py:161} INFO - Started process (PID=7721) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:09:15.322+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:09:15.323+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:09:15.323+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:09:15.629+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:09:15.627+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 8, in <module>
    from airflow.providers.python.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.python'
[2024-03-18T08:09:15.630+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:09:15.648+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.331 seconds
[2024-03-18T08:09:57.460+0000] {processor.py:161} INFO - Started process (PID=8004) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:09:57.462+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:09:57.463+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:09:57.463+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:09:57.768+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:09:57.766+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 8, in <module>
    from airflow.providers.python.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.python'
[2024-03-18T08:09:57.768+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:09:57.789+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.333 seconds
[2024-03-18T08:10:38.740+0000] {processor.py:161} INFO - Started process (PID=8310) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:10:38.742+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:10:38.743+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:10:38.742+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:10:39.049+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:10:39.047+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 8, in <module>
    from airflow.providers.python.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.python'
[2024-03-18T08:10:39.049+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:10:39.071+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.335 seconds
[2024-03-18T08:10:56.033+0000] {processor.py:161} INFO - Started process (PID=8423) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:10:56.035+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:10:56.036+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:10:56.035+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:10:56.352+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:10:56.349+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 93, in <module>
    transform_load_weather_data = PythonOperator(
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 420, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-03-18T08:10:56.353+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:10:56.383+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.353 seconds
[2024-03-18T08:11:40.007+0000] {processor.py:161} INFO - Started process (PID=8887) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:11:40.009+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:11:40.010+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:11:40.010+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:11:40.318+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:11:40.317+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 93, in <module>
    transform_load_weather_data = PythonOperator(
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 420, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-03-18T08:11:40.319+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:11:40.338+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.335 seconds
[2024-03-18T08:11:59.481+0000] {processor.py:161} INFO - Started process (PID=8968) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:11:59.483+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:11:59.484+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:11:59.483+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:11:59.783+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:11:59.782+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 93, in <module>
    transform_load_weather_data = PythonOperator(
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 420, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-03-18T08:11:59.784+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:11:59.810+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.332 seconds
[2024-03-18T08:12:19.738+0000] {processor.py:161} INFO - Started process (PID=9047) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:12:19.740+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:12:19.741+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:12:19.741+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:12:20.076+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:12:20.075+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 93, in <module>
    transform_load_weather_data = PythonOperator(
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 420, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-03-18T08:12:20.077+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:12:20.126+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.391 seconds
[2024-03-18T08:13:02.812+0000] {processor.py:161} INFO - Started process (PID=9390) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:13:02.818+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:13:02.819+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:13:02.819+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:13:04.015+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:13:04.014+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 93, in <module>
    transform_load_weather_data = (
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 420, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-03-18T08:13:04.016+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:13:04.102+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 1.302 seconds
[2024-03-18T08:13:26.151+0000] {processor.py:161} INFO - Started process (PID=9532) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:13:26.152+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:13:26.153+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:13:26.153+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:13:26.451+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:13:26.450+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 93, in <module>
    transform_load_weather_data = PythonOperator(
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 420, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-03-18T08:13:26.451+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:13:26.471+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.324 seconds
[2024-03-18T08:14:05.957+0000] {processor.py:161} INFO - Started process (PID=9693) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:14:05.959+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:14:05.960+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:14:05.960+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:14:06.291+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:14:06.290+0000] {dagbag.py:350} ERROR - Failed to import: /home/ubuntu/airflow/dags/w2.py
Traceback (most recent call last):
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/w2.py", line 93, in <module>
    transform_load_weather_data = PythonOperator(
  File "/home/ubuntu/air_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 420, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-03-18T08:14:06.292+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:14:06.313+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.359 seconds
[2024-03-18T08:14:23.988+0000] {processor.py:161} INFO - Started process (PID=9772) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:14:23.989+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:14:23.990+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:14:23.990+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:14:24.302+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:14:24.568+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:14:24.567+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:14:24.587+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:14:24.587+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-17 00:00:00+00:00, run_after=2024-03-18 00:00:00+00:00
[2024-03-18T08:14:24.621+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.637 seconds
[2024-03-18T08:15:18.314+0000] {processor.py:161} INFO - Started process (PID=10309) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:15:18.316+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:15:18.317+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:15:18.317+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:15:18.639+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:15:18.991+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:15:18.982+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:15:19.051+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:15:19.051+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:15:19.124+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.814 seconds
[2024-03-18T08:16:12.899+0000] {processor.py:161} INFO - Started process (PID=11023) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:16:12.901+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:16:12.902+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:16:12.901+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:16:13.214+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:16:13.234+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:16:13.233+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:16:13.260+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:16:13.259+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:16:13.284+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.389 seconds
[2024-03-18T08:16:59.065+0000] {processor.py:161} INFO - Started process (PID=11549) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:16:59.067+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:16:59.068+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:16:59.068+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:16:59.369+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:16:59.385+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:16:59.384+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:16:59.414+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:16:59.414+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:16:59.666+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.605 seconds
[2024-03-18T08:17:58.681+0000] {processor.py:161} INFO - Started process (PID=12330) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:17:58.683+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:17:58.684+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:17:58.684+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:17:59.002+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:17:59.017+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:17:59.017+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:17:59.040+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:17:59.040+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:17:59.064+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.386 seconds
[2024-03-18T08:18:32.925+0000] {processor.py:161} INFO - Started process (PID=12485) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:18:32.927+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:18:32.928+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:18:32.927+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:18:33.240+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:18:33.255+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:18:33.254+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:18:33.430+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:18:33.430+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:18:33.454+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.532 seconds
[2024-03-18T08:19:18.439+0000] {processor.py:161} INFO - Started process (PID=12707) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:19:18.441+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:19:18.442+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:19:18.442+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:19:18.744+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:19:18.760+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:19:18.759+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:19:18.790+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:19:18.790+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:19:18.814+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.378 seconds
[2024-03-18T08:19:49.434+0000] {processor.py:161} INFO - Started process (PID=12977) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:19:49.435+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:19:49.437+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:19:49.436+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:19:49.736+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:19:49.751+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:19:49.750+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:19:49.774+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:19:49.774+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:19:49.960+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.530 seconds
[2024-03-18T08:20:21.306+0000] {processor.py:161} INFO - Started process (PID=13247) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:20:21.308+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:20:21.309+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:20:21.308+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:20:21.634+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:20:21.803+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:20:21.802+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:20:21.824+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:20:21.824+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:20:21.848+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.546 seconds
[2024-03-18T08:21:17.620+0000] {processor.py:161} INFO - Started process (PID=13600) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:21:17.622+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:21:17.623+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:21:17.622+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:21:17.942+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:21:17.958+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:21:17.957+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:21:17.982+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:21:17.982+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:21:18.150+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.534 seconds
[2024-03-18T08:22:06.826+0000] {processor.py:161} INFO - Started process (PID=13771) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:22:06.827+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:22:06.828+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:22:06.828+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:22:07.188+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:22:07.203+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:22:07.202+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:22:07.224+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:22:07.224+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:22:07.252+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.430 seconds
[2024-03-18T08:22:46.012+0000] {processor.py:161} INFO - Started process (PID=14055) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:22:46.014+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:22:46.015+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:22:46.014+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:22:46.308+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:22:46.324+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:22:46.323+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:22:46.506+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:22:46.505+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:22:46.529+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.521 seconds
[2024-03-18T08:23:23.964+0000] {processor.py:161} INFO - Started process (PID=14382) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:23:23.965+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:23:23.966+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:23:23.966+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:23:24.274+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:23:24.289+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:23:24.289+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:23:24.337+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:23:24.337+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:23:24.364+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.404 seconds
[2024-03-18T08:24:05.271+0000] {processor.py:161} INFO - Started process (PID=14673) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:24:05.273+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:24:05.274+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:24:05.274+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:24:05.590+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:24:05.607+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:24:05.607+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:24:05.807+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:24:05.806+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:24:05.841+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.574 seconds
[2024-03-18T08:24:43.504+0000] {processor.py:161} INFO - Started process (PID=15003) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:24:43.505+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:24:43.507+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:24:43.506+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:24:43.817+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:24:43.833+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:24:43.832+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:24:43.856+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:24:43.856+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:24:44.049+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.549 seconds
[2024-03-18T08:25:25.279+0000] {processor.py:161} INFO - Started process (PID=15284) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:25:25.281+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:25:25.282+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:25:25.282+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:25:25.590+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:25:25.613+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:25:25.612+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:25:25.792+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:25:25.791+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:25:25.815+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.539 seconds
[2024-03-18T08:26:09.231+0000] {processor.py:161} INFO - Started process (PID=15387) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:26:09.233+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:26:09.234+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:26:09.233+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:26:09.701+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:26:09.713+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:26:09.712+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:26:09.734+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:26:09.733+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:26:09.758+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.530 seconds
[2024-03-18T08:26:49.526+0000] {processor.py:161} INFO - Started process (PID=15487) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:26:49.528+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:26:49.529+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:26:49.529+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:26:49.850+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:26:50.021+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:26:50.020+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:26:50.044+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:26:50.043+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:26:50.068+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.545 seconds
[2024-03-18T08:27:31.347+0000] {processor.py:161} INFO - Started process (PID=15529) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:27:31.349+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:27:31.350+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:27:31.349+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:27:31.805+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:27:31.818+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:27:31.817+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:27:31.840+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:27:31.840+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:27:31.865+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.521 seconds
[2024-03-18T08:28:13.053+0000] {processor.py:161} INFO - Started process (PID=15629) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:28:13.055+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:28:13.056+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:28:13.055+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:28:13.511+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:28:13.525+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:28:13.524+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:28:13.545+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:28:13.545+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:28:13.569+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.520 seconds
[2024-03-18T08:28:54.961+0000] {processor.py:161} INFO - Started process (PID=15730) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:28:54.963+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:28:54.964+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:28:54.963+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:28:55.407+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:28:55.419+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:28:55.418+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:28:55.438+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:28:55.438+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:28:55.463+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.506 seconds
[2024-03-18T08:29:38.992+0000] {processor.py:161} INFO - Started process (PID=15772) to work on /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:29:38.994+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/w2.py for tasks to queue
[2024-03-18T08:29:38.995+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:29:38.995+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:29:39.441+0000] {processor.py:840} INFO - DAG(s) 'weather_dag' retrieved from /home/ubuntu/airflow/dags/w2.py
[2024-03-18T08:29:39.454+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:29:39.454+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-18T08:29:39.474+0000] {logging_mixin.py:188} INFO - [2024-03-18T08:29:39.474+0000] {dag.py:3834} INFO - Setting next_dagrun for weather_dag to 2024-03-18 00:00:00+00:00, run_after=2024-03-19 00:00:00+00:00
[2024-03-18T08:29:39.497+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/w2.py took 0.508 seconds
